application-activedirectory-ui/src/main/resources/ActiveDirectory/Code/Main.xml

```diff
@@ -342,10 +364,11 @@
           'resultsParameter': 'results',
           'resultValue': 'value'
         });
-         var aDInput = $('&lt;input/&gt;').attr({
+        var unescapedADValue = unescape(aDValue);
```

Comment:
Is this variable used anywhere else? If not then why not set directly the value attribute?